docs/quick-start.md

```diff
@@ -720,7 +733,7 @@ const keyring = new Keyring({ type: 'sr25519' });
 
 async function createApi() {
   // Initialise the provider to connect to the local node
-  const provider = new WsProvider('wss://testnet.polygonavail.net/ws');
+  const provider = new WsProvider('ws://127.0.0.1:9944');
```

Comment:
This points to the local endpoint. Do we expect people to run a node locally in the quick start guide?